packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 48.16.0 - May 20, 2020
+
+## Added
+
+#### salesforcedx-vscode-code
+
+- Disabling telemetry in the extensions also disables telemetry collected in the CLI, and vice versa ([PR #2193](https://github.com/forcedotcom/salesforcedx-vscode/pull/2193))
+
```

Comment:
We should add https://github.com/forcedotcom/salesforcedx-vscode/pull/2103 too since it fixes a bug that's logged.